src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -114,8 +114,8 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
         return cosmosChainAdapter.getFeeData({})
       }
       case ChainTypes.Osmosis: {
-        // TODO(gomes): implement Osmosis support
-        return {} as chainAdapters.FeeDataEstimate<ChainTypes>
+        const osmosisChainAdapter = await chainAdapterManager.byChainId('cosmos:osmosis-1')
```

Comment:
Do we have a way to get the chainId without hard coding? If we have a good reference point, might be worth it to update these strings to that reference.

Reply:
We can follow up once https://github.com/shapeshift/lib/pull/676/files is in place